### ISSUE TITLE
ReactionImages - "upload" not a function

### ISSUE THREAD

**Issue by alphaqueueop** [Metalloriff/BetterDiscordPlugins#78]
```
Uncaught TypeError: NeatoLib.Modules.get(...).upload is not a function
    at HTMLImageElement.images.(anonymous function).addEventListener.e (X:\Users\%username%\AppData\Roaming\BetterDiscord\plugins\ReactionImages.plugin.js:168:55)
    at HTMLImageElement.i (456d414469fa485a824e.js:49)
```

**Comment by Patchumz** [comment]
Another instance, with SendLinksDirectly

`Uncaught TypeError: NeatoLib.Modules.get(...).upload is not a function
    at NeatoLib.requestFile.file (C:\Users\%user%\AppData\Roaming\BetterDiscord\plugins\SendLinksDirectly.plugin.js:77)
    at IncomingMessage.x.on (NeatoBurritoLibrary.js:2311)
    at IncomingMessage.emit (events.js:187)
    at endReadableNT (_stream_readable.js:1092)
    at process._tickCallback (internal/process/next_tick.js:63)
NeatoLib.requestFile.file @ C:\Users\Patch\AppData\Roaming\BetterDiscord\plugins\SendLinksDirectly.plugin.js:77
x.on @ NeatoBurritoLibrary.js:2311
emit @ events.js:187
endReadableNT @ _stream_readable.js:1092
_tickCallback @ internal/process/next_tick.js:63`

**Comment by Kamisoi** [comment]
https://github.com/Metalloriff/BetterDiscordPlugins/pull/89

**Comment by l0c4lh057** [comment]
Closed in #89
